src/VisualStudio/Core/Def/LanguageService/AbstractLanguageService`2.cs

```diff
@@ -130,46 +132,6 @@ internal void TearDown()
         }
     }
 
-    protected virtual void GetServices()
-    {
-        // This method should only contain calls to acquire services off of the component model
-        // or service providers.  Anything else which is more complicated should go in Initialize
-        // instead.
-        this.EditorOptionsService = this.Package.ComponentModel.GetService<EditorOptionsService>();
-        this.Workspace = this.Package.ComponentModel.GetService<VisualStudioWorkspaceImpl>();
-        this.EditorAdaptersFactoryService = this.Package.ComponentModel.GetService<IVsEditorAdaptersFactoryService>();
-        this.AnalyzerFileWatcherService = this.Package.ComponentModel.GetService<AnalyzerFileWatcherService>();
-    }
-
-    protected virtual void RemoveServices()
-    {
-        this.EditorAdaptersFactoryService = null;
-        this.Workspace = null;
-    }
```

Comment:
this doesn't seem to serve any purpose.  nulling out these simple managed types serves no purpose.